app/models/npq_lead_provider.rb

```diff
@@ -3,6 +3,6 @@
 class NPQLeadProvider < ApplicationRecord
   belongs_to :cpd_lead_provider, optional: true
 
-  has_many :npq_profiles, class_name: "NPQValidationData"
+  has_many :npq_profiles, class_name: "NPQApplication"
```

Comment:
That one as well is confusing. However maybe let's do it in its own separate little PR.

Profiles are the `ParticipantProfile::(ECF||NPQ)` 

@kicferk1 would you mind creating a ticket in the backlog please?

Reply:
I might spend some time in the afternoon fixing it :D 

I think https://dfedigital.atlassian.net/browse/CPDTP-538 mentions this too, but I agree, separate PR for this is a good idea :)